views/partials/footer.ejs

```diff
@@ -0,0 +1,20 @@
+<div class="row">
+
+    <div class="col" style="text-align:center;">
+
+        <footer>
+
+            <hr>
+
+            <svg xmlns="http://www.w3.org/2000/svg" width="32" height="32" fill="grey" class="bi bi-github"

```

Comment:
Remove the following svg path and correctly include bootrap icons so that only the class bi-github is needed